sovtoken/sovtoken/token_req_handler.py

```diff
@@ -287,7 +287,12 @@ def __commit__(utxo_cache, ledger, state, txnCount, stateRoot, txnRoot,
     def _commit_to_utxo_cache(utxo_cache, state_root):
         state_root = base58.b58decode(state_root.encode()) if isinstance(
             state_root, str) else state_root
-        assert utxo_cache.first_batch_idr == state_root
+        if utxo_cache.first_batch_idr != state_root:
+            raise TokenValueError(
+                'state_root', state_root,
+                ("equal to utxo_cache.first_batch_idr hash {}"
+                 .format(utxo_cache.first_batch_idr))
+            )
```

Comment:
Are these changes coming from master? @KitHat

Reply:
Yes, but I should've cleaned off them. Strange.